log_processor.py

```diff
@@ -279,7 +297,10 @@ def parse_line(self, row):
             player_amounts = {player: stack_size for (player, stack_size) in zip(players, stack_size_counts)}
             for player, amount in player_amounts.items():
                 if amount != self.evening.players[player]:
-                    print(f"**WARNING** {player}: {amount} != {self.evening.players[player]}")
+                    round_no = self._current_round.number
+                    print(f"**WARNING** start of round #{round_no}: "
+                          f"{player}: {amount} (amount from log) != {self.evening.players[player]} (our amount)")
+                    print("winners in prev round: ", self.evening.rounds[-2].winners)
```

Comment:
I like the extra logging. The `rounds[-2]` does break on the log file provided by #2 though. Please handle the case when there are no previous winners.

Reply:
Sure, added an extra check.